wnbd-client/client.cpp

```diff
@@ -352,4 +373,11 @@ Client::Command commands[] = {
     Client::Command(
         "reset-opt", {}, "Reset driver option.",
         execute_reset_opt, get_reset_opt_args),
+    Client::Command(
+        "install-driver", {}, "Install WNBD driver and create its adapter",
+        execute_install, get_filename_args),
+    Client::Command(
+        "clean-uninstall", {}, "Remove all disk mappings with a HARD disconnect option"
```

Comment:
nit: I'm not a big fan of "clean-uninstall", wondering if we should stick to "uninstall-driver". It's quite obvious that the existing mappings might be removed when the driver is uninstalled.

Also, "Remove all disk mappings with a HARD disconnect option" sounds a bit weird, how about "Hard remove all disk mappings and adapters and uninstall all WNBD driver instances".